deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,382 @@
+        if value not in artillery_result_type:
+            msg = f'The Artillery result type to generate the CSV is not ' \
+                'valid. Accepted Values: {artillery_result_type}. ' \
+                'Value received {value}.'
+
+            raise Exception(msg)
+
+
+def format_directory(directory: str) -> str:
+    """Check the paths of the directories and format them if necessary.
+
+    Args:
+        args (str): Path of a directory.
+
+    Returns:
+        str: Formatted path.
+    """
+    return join(directory, '')
+
+
+def create_directories(directory: str) -> None:
+    """Create directory that does not exist.
+
+    Args:
+        directory (str): Path of directory.
+    """
+    makedirs(directory, exist_ok=True)
+
+
+def provide_directories(args: argparse) -> None:
+    """Format the paths of the directories and create them
+    (if they do not exist).
+
+    Args:
+        args (argparse): Script parameters.
+    """
+    # Format Directories
+    args.logs = format_directory(args.logs)
+    args.screenshots = format_directory(args.screenshots)
+    args.csv = format_directory(args.csv)
+    args.session = format_directory(args.session)
+
+    # Create Directories
+    create_directories(args.logs)
+    create_directories(args.screenshots)
+    create_directories(args.csv)
+    create_directories(args.session)
+
+
+def create_session_file(path: str, user: str) -> None:
+    """Create file to save the browser session.
+
+    Args:
+        path (str): Path of directory.
+        user (str): Username.
+    """
+    file_path = f"{path}user-{user}.json"
+
+    with open(file_path, 'w') as file:
+        file.write('{}')
+
+
+def gen_artillery_params(args: argparse) -> dict:
+    """Format all parameters for Artillery.
+
+    Args:
+        args (argparse): Script parameters.
+
+    Returns:
+        dict: Artillery parameters.
+    """
+    artillery_params = {
+        'username': args.user,
+        'password': args.password,
+        'screenshots': args.screenshots,
+        'session': args.session
+    }
+
+    return artillery_params
+
+
+def add_options_to_artillery(config: str, options: dict) -> None:
+    """Add missing parameters to Artillery config.
+
+    Args:
+        config (str): YAML de Artillery.
+        options (dict): Parametros de Artillery.
+    """
+    # Open YML
+    with open(config) as f:
+        data = yaml.safe_load(f)
+
+    # Check `config` in Artillery YAML
+    if 'config' not in data.keys():
+        data['config'] = {}
+
+    # Check `variables` in Artillery YAML
+    if 'variables' not in data['config'].keys():
+        data['config']['variables'] = {}
+
+    # Add Options to Artillery
+    for key in options:
+        if key not in data['config']['variables'].keys():
+            data['config']['variables'][key] = f'{{{{ { key } }}}}'
+
+    # Save YML
+    with open(config, "w") as f:
+        yaml.dump(data, f)
+
+
+def process_script_arguments(args: argparse) -> None:
```

Comment:
Same here, use Namespace

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/a07ebb3ed73c7bc3ab8fce58bf98032b832575f8